### PR TITLE
Deploy 20230622

### PR DESCRIPTION
**Pull request by m-ball3**
Updated recovery field data in field_records for 20230622, updated deployment field data in field_records for 20230622, uploaded field pics 20230622 deployment.

@leewujung @aditya-uw @ctuguinay

### CODE REVIEW

**Comment by aditya-uw** [comment]
Hi @ctuguinay @m-ball3, I was looking into the `recover-20230622` folders and I saw that in UBNA_007 there were files from 03:00 to 13:00 UTC of 658 MB and in UBNA_008 there were files from 00:00 to 24:00 UTC of 165 MB.

This is the opposite of what you've put into the field records I think because in the records, UBNA_007 belonged to a daytime recorder and UBNA_008 belonged to a nighttime recorder. Could you look this over and correct it? Thanks! The other folders seem fine!

File: field_records/ubna_2023.csv
Comment (ctuguinay) [inline field_records/ubna_2023.csv:20]: Will need to change this to the appropriate nighttime values.

File: field_records/ubna_2023.csv
Comment (ctuguinay) [inline field_records/ubna_2023.csv:21]: Will need to change this to the appropriate all day values.

File: field_records/ubna_2023.csv
Comment (ctuguinay) [inline field_records/ubna_2023.csv:26]: Will need to change this to the appropriate night time values.

File: field_records/ubna_2023.csv
Comment (ctuguinay) [inline field_records/ubna_2023.csv:27]: Will need to change this to the appropriate all day values.

**Review by ctuguinay — CHANGES_REQUESTED** [review]
@m-ball3 Please read Aditya's previous comment. This was my bad and I should probably be the one making these changes since I was the scribe that initially marked the wrong configurations, but I think revising your code and updating your pull request in this scenario will be good practice.

Please follow the following steps to revise your pull request accordingly. These steps contain a lot of the things that you did yesterday in the office in your initial pull request:
1. Make sure you are on the correct branch using `git branch`. It should indicate that you are in the `deploy-20230622` branch.
2. Look through the comments that both Aditya and I made in your pull request. Please ping us if you're unable to see them and we'll direct you to them. They should be available to you where you are viewing this longer comment: [https://github.com/uw-echospace/union-bay-bats/pull/84](https://github.com/uw-echospace/union-bay-bats/pull/84)
3. Make the appropriate changes based on our comments.
4. Run the pytest command: `python -m pytest`.
5. Manually change your directory in your terminal using the `cd (insert command)` commands to get to your changed file or files and use `git add (filename)`.
6. Once you have run `git status` to check which files you have added. If these look good to you proceed. If not, run `git reset` and repeat step 5 again.
7. Use `git commit -m "your commit message"` to commit your changes and add a message for your changes. This commit message can basically be what you did in your changes.
8. Run `git push` (again, make sure that you are in the `deploy-20230622` branch and not in the `main` branch.
9. Check your changes in [https://github.com/uw-echospace/union-bay-bats/pull/84](https://github.com/uw-echospace/union-bay-bats/pull/84). If what you see satisfies the comments made on your pull request, please ping both Aditya and me.

You may have to repeat this process if any other comments are made, but hopefully not.

File: field_records/ubna_2023.csv
Comment (ctuguinay) [inline field_records/ubna_2023.csv:20]: Will need to change this to the appropriate night configuration values.

File: field_records/ubna_2023.csv
Comment (ctuguinay) [inline field_records/ubna_2023.csv:21]: Will need to change this to the appropriate all-day configuration values.

File: field_records/ubna_2023.csv
Comment (ctuguinay) [inline field_records/ubna_2023.csv:26]: Will need to change this to the appropriate all-day configuration values.

File: field_records/ubna_2023.csv
Comment (ctuguinay) [inline field_records/ubna_2023.csv:27]: Will need to change this to the appropriate all-day configuration values.

File: field_records/ubna_2023.csv
Comment (ctuguinay) [reply field_records/ubna_2023.csv:26]: Remove the `Changed from daytime to nighttime might need to change previous values`. With the change above, this will be resolved.

File: field_records/ubna_2023.csv
Comment (ctuguinay) [reply field_records/ubna_2023.csv:27]: Please remove the `Changed from daytime to nighttime might need to change previous values`. With the change above, this will be resolved.

**Comment by m-ball3** [comment]
Hi @aditya-uw and @ctuguinay, I made the changes requested! Please let me know if there are any issues with what I did and I will fix them. (:

**Comment by aditya-uw** [comment]
Hi @m-ball3! You're doing good! I have a couple things:
- I think it's good practice to limit the characters you use in your git commit messages. See [here](https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/). I see that your git commit messages are lengthy and this can confuse you as to what exactly you changed with a commit as you make more and more of them. If you feel like the message itself cannot be reduced in length because you worked on several things, consider making each of those several things be committed with their own message :) 
- @ctuguinay, your changes imply UBNA_007 was inside Audiomoth A and UBNA_008 was inside Audiomoth B, is that correct? 
   - I believe Audiomoth A has been set aside as a daytime recorder and Audiomoth B has been set aside as a nighttime recorder. So the changes you suggested seem to imply that you made Audiomoth A a nighttime recorder and vice versa? I initially thought you just mis-entered which Audiomoths UBNA_007 and UBNA_008 were inside. Maybe UBNA_007 was in Audiomoth B and UBNA_008 was in Audiomoth A?

**Comment by aditya-uw** [comment]
@m-ball3 Also feel free to snoop around and look at [previous Pull Requests](https://github.com/uw-echospace/union-bay-bats/pulls?q=is%3Apr+is%3Aclosed). We admittedly aren't perfect either but it's the journey XD

**Comment by ctuguinay** [comment]
@aditya-uw It's either the case that you described where we miswrote 007 and 008 or we have A and B in the wrong configurations on the field records. My bad for the misunderstanding. I think what we should do is maybe put this on hold and I'll bring my laptop tomorrow to check the configurations in person. I think you're correct though, and we'll update this accordingly.

@m-ball3 Also, I kind of got the git commit command wrong for writing the description and such. `git commit -m "Title" -m "Description ..........";`, where the title is short and the description is longer. I do realize that yesterday I told you to write as much as you could and that was my bad too. Let's try to keep them more concise, but in the case where you cannot keep the description to something short while describing everything you've done, I'm of the opinion that it's better to write more than leave any large details out. Great job on the first pull request though! This may have been the most complicated first pull request ever lol.

**Comment by ctuguinay** [comment]
This looks good to me. What do you think @aditya-uw? I'll also send in another PR for this current day's PR once this has been merged.

**Comment by aditya-uw** [comment]
This looks good @ctuguinay and @m-ball3! Feel free to squash and merge!

**Pull request merged**
